internal/config/tiers.go

```diff
@@ -205,6 +205,9 @@ var TierBronzeConfiguration = Lintroller{
 	Why: Why{
 		Enabled: false,
 	},
+	Errorlint: Errorlint{
+		Enabled: false,
```

Comment:
I would warn here, too. I think the eventual target level for this linter is Bronze.